### PR TITLE
3.X API installation update

### PR DESCRIPTION
**Pull request by chensuyue**
## Type of Change

others

## Description

- [x] 3.X API installation update
- [x] Remove 3.X ORT

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,21 +19,24 @@ Intel® Neural Compressor aims to provide popular model compression techniques s
...
 
 ## What's New
+* [2024/07] From 3.0 release, framework extension API is recommended to be used for LLM quantization.
```
Comment (yiliu30) [inline README.md:29]: 1) I think `for LLM quantization` is not accurate, we provide the same APIs for all supported model.s
2) How about adding a link for more details?

File: docs/source/installation_guide.md [hunk 2]
```diff
@@ -76,10 +75,15 @@ The AI Kit is distributed through many common channels, including from Intel's w
 ## System Requirements
 
 ### Validated Hardware Environment
+
+#### Intel® Neural Compressor supports HPUs based on xxx
+* Intel Gaudi Al Accelerators
```
Comment (yiliu30) [inline docs/source/installation_guide.md:80]: Gaudi 2?

**Review by yiliu30 — APPROVED** [review]
left some comments, others are LTGM.

File: README.md [hunk 3]
```diff
@@ -19,21 +19,24 @@ Intel® Neural Compressor aims to provide popular model compression techniques s
...
+* Support a wide range of Intel hardware such as [Intel Gaudi Al Accelerators](https://www.intel.com/content/www/us/en/products/details/processors/ai-accelerators/gaudi-overview.html), [Intel Core Ultra Processors](https://www.intel.com/content/www/us/en/products/details/processors/core-ultra.html), [Intel Xeon Scalable Processors](https://www.intel.com/content/www/us/en/products/details/processors/xeon/scalable.html), [Intel Xeon CPU Max Series](https://www.intel.com/content/www/us/en/products/details/processors/xeon/max-series.html), [Intel Data Center GPU Flex Series](https://www.intel.com/content/www/us/en/products/details/discrete-gpus/data-center-gpu/flex-series.html), and [Intel Data Center GPU Max Series](https://www.intel.com/content/www/us/en/products/details/discrete-gpus/data-center-gpu/max-series.html) with extensive testing; support AMD CPU, ARM CPU, and NVidia GPU through ONNX Runtime with limited testing
```
[the remark this answers is not in the document]
Comment (chensuyue) [reply README.md:22]: 1. Yes, 2x ORT still support. 
2. Ok, I will update.

File: README.md (hunk 1 above)
Comment (chensuyue) [reply README.md:29]: 1. Yes, I will update with quantization. 
2. link can be found in matrix, I think we don't need to expand here.

File: docs/source/installation_guide.md (hunk 2 above)
Comment (chensuyue) [reply docs/source/installation_guide.md:80]: will update

File: README.md [hunk 4]
```diff
@@ -19,21 +19,25 @@ Intel® Neural Compressor aims to provide popular model compression techniques s
...
+* Support a wide range of Intel hardware such as [Intel Gaudi Al Accelerators](https://www.intel.com/content/www/us/en/products/details/processors/ai-accelerators/gaudi-overview.html), [Intel Core Ultra Processors](https://www.intel.com/content/www/us/en/products/details/processors/core-ultra.html), [Intel Xeon Scalable Processors](https://www.intel.com/content/www/us/en/products/details/processors/xeon/scalable.html), [Intel Xeon CPU Max Series](https://www.intel.com/content/www/us/en/products/details/processors/xeon/max-series.html), [Intel Data Center GPU Flex Series](https://www.intel.com/content/www/us/en/products/details/discrete-gpus/data-center-gpu/flex-series.html), and [Intel Data Center GPU Max Series](https://www.intel.com/content/www/us/en/products/details/discrete-gpus/data-center-gpu/max-series.html) with extensive testing; 
+support AMD CPU, ARM CPU, and NVidia GPU through ONNX Runtime with limited testing; support NVidia GPU for some WOQ algos like auto-round, hqq.
```
Comment (thuang6) [inline README.md:23]: suggest change "algos like auto-round, hqq." to "algorithms like AutoRound and HQQ" so that we have a consistent term in doc

**Review by thuang6 — APPROVED** [review]

**Comment by mengniwang95** [comment]
LGTM

File: README.md (hunk 4 above)
Comment (chensuyue) [reply README.md]: updated

**Comment by chensuyue** [comment]
ITREX UT known issue, fix WIP.

**Pull request merged**